internal/pkg/deploy/pipeline.go

```diff
@@ -132,11 +134,21 @@ func PipelineSourceFromManifest(mfSource *manifest.Source) (source interface{},
 				PersonalAccessTokenSecretID: (mfSource.Properties["access_token_secret"]).(string),
 			}, false, nil
 		} else {
+			// If an existing CSC connection is being used, don't prompt to update connection from 'PENDING' to 'AVAILABLE'.
+			var shouldPrompt bool
+			var connection string
+			if (mfSource.Properties["connection_ARN"]) == nil {
```

Comment:
instead of the ARN can we use the connection name? since the connection needs to be defined in the same account and region as the pipeline

In our manifest, I think it would be neat to have `connection_name` as a field instead of `connection_ARN` (folks don't like entering full ARNs in the manifests since it has the account ID in it). 
> side note: in our manifests let's not capitalize any of the letters and stick to lower_snake_case

Reply:
The parenthesis seems off here, what do you think of doing this instead?
```go
connection, ok := mfSource.Properties["connection_name"]
repo := &GitHubSource{
	ProviderName:  manifest.GithubProviderName,
	Branch:        (mfSource.Properties["branch"]).(string),
	RepositoryURL: GitHubURL((mfSource.Properties["repository"]).(string)),
}
if !ok {
   return repo, true, nil
}
repo.ConnectionName = connection.(string) 
return repo, false, nil
```

Reply:
I'm not sure we can get the connection ARNs from just the connection name. `GetConnection` requires the ARN for all the other info.